samza-core/src/main/scala/org/apache/samza/metrics/reporter/Metrics.scala

```diff
@@ -52,4 +54,8 @@ class Metrics(metrics: Map[String, Map[String, Object]]) {
   def get(group: String) = immutableMetrics.get(group)
 
   def getAsMap(): Map[String, Map[String, Object]] = Collections.unmodifiableMap(immutableMetrics)
+
+  def this() {
```

Comment:
I did a code search and couldn't find any reference to this default constructor. What's the reason to add it?